### DISCUSSION TITLE
Focus last tab map command?

### DISCUSSION THREAD

**Discussion by chmouel** [kovidgoyal/kitty#6242] [General]
I am trying to focus my last focused tab with a shortcut, this works well: 

```
kitty @ focus-tab -m recent:1 
```

but i can't figure out how to do it directly in a map keybinding, like:

```
map kitty_mod+space        focus-tab -m recent:1 
```

none of the actions here match this https://sw.kovidgoyal.net/kitty/actions/

**Comment by kovidgoyal** [comment]
See https://sw.kovidgoyal.net/kitty/remote-control/#mapping-key-presses-to-remote-control-commands
